src/Command/AddMassMediaCommand.php

```diff
@@ -61,11 +61,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $fp = $this->getFilePointer($input, $output);
 
         $delimiter = $input->getOption('delimiter');
-        \assert(null === $delimiter || is_string($delimiter));
+        \assert(\is_string($delimiter));
```

Comment:
I think it's better to add this to the baseline rather than doing those always true assert.
When we'll configure correctly phpstan, the baseline will be reported as useless.

And what do you think about opening an issue to the phpstan-symfony repository ? :)

Reply:
Yep, I just wanted to get a green build first. 

I don't really understand why it doesn't detect it. But to be honest I don't understand how phpstan can know if the parameter will be an array or an string, do you know the internals of this? There are some parameters that we provide a default value, so it could know because of that, but we have a lot of parameters without default value.

I will try to open an issue on phpstan-symfony

Reply:
Sure, it's done by this dynamic return extension:
https://github.com/phpstan/phpstan-symfony/blob/master/src/Type/Symfony/InputInterfaceGetOptionDynamicReturnTypeExtension.php
especially this code:
https://github.com/phpstan/phpstan-symfony/blob/master/src/Type/Symfony/InputInterfaceGetOptionDynamicReturnTypeExtension.php#L65-L79
It look at the option definition to find out his type.

Reply:
I found the problem.

The commands are registered as LazyCommand because we define the title and description on a static variable.

This does not happen on SonataAdmin (missing description).

Reply:
https://github.com/phpstan/phpstan-symfony/issues/185 Here is the issue and I tried to fix it too with a PR there.

Ideally all Sonata commands should be as lazy as possible (they should still do the work 😄 )

Reply:
Fixed, phpstan will pass once the phpstan/phpstan-symfony#185 is solved.